app/src/main/kotlin/com/bael/dads/di/module/MainActivityModule.kt

```diff
@@ -0,0 +1,22 @@
+package com.bael.dads.di.module
+
+import com.bael.dads.activity.MainActivity
+import com.bael.dads.lib.presentation.di.ActivityNameQualifier
+import com.bael.dads.lib.presentation.di.ActivityNameQualifier.Companion.ACTIVITY_MAIN
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
```

Comment:
Could you please put the author's information?

Reply:
Actually I don't set that author's information in my IDE, but ok

Reply:
Actually I don't set that author's information in my IDE, but ok